src/common/api/base.go

```diff
@@ -112,6 +112,7 @@ func (b *BaseAPI) Redirect(statusCode int, resouceID string) {
 	requestURI := b.Ctx.Request.RequestURI
 	resourceURI := requestURI + "/" + resouceID
 
+	b.Ctx.ResponseWriter.Header().Set("Content-Type", "application/json")
```

Comment:
I'd like to modify the robot api only, not for the base method.

Reply:
Okey i will change it as you mentioned.

Reply:
thanks, as we're closing to release, is it possible to finish it by the end next Wed?

Reply:
yes i will do it.

Reply:
Done. @wy65701436